src/main/scala/devices/gpio/GPIO.scala

```diff
@@ -28,7 +28,7 @@ import sifive.blocks.util.{DeviceParams,DeviceAttachParams,BasicBusBlocker}
 // we could do the pinmux inside.
 
 class GPIOPortIO(val c: GPIOParams) extends Bundle {
-  val pins = Vec(c.width, new EnhancedPin())
+  val pins = Vec(c.width, new LN14LPPEnhancedPin())
```

Comment:
Have you made changes to the GPIO device to drive these new signals? Or are they dangling?

Reply:
The intention is to connect these signals. There are more works to be done. The registers in current GPIO don't match all the io signals.

Reply:
I have added additional registers to drive those new signals.